contracts/contracts/Governance.sol

```diff
@@ -77,14 +77,19 @@ contract Governance {
         return validators[_sender];
     }
 
-    // Check if token is known
+    // Fail if token is unknown
     function requireValidTokenId(uint16 _tokenId) external view {
         require(
             _tokenId < totalTokens + 1,
             "grd11"
         ); // grd11 - unknown token id
     }
 
+    // Check if token is known
```

Comment:
one of the functions is redundant, I suspect this one -- see comment above.

Reply:
@dvush pls fix it and merge as soon as possible, so I could merge my PRs based on yours

Reply:
I am not sure that it should be fixed, see my comment above.

Reply:
Then please remove `requireValidTokenId()` in favor of `require(isValidTokenId(_token))`.

No logic duplication in contract.

Reply:
I've already removed logic duplication [1ead02](https://github.com/matter-labs/zksync/pull/215/commits/1ead026e1670043d9f64e54c40924038e9349b09)

`requireValidTokenId(_token) === require(isValidTokenId(_token))` 

It is just a name for convenience and this way we have error message defined in one place.

Reply:
ok